### DISCUSSION TITLE
how to build for net8.0

### DISCUSSION THREAD

**Discussion by corsiva** [dotnet/aspnetcore#52833] [Q&A] [answered]
Hello,

I can build aspnetcore but it's built for net9.0.
I want to build it for net8.0.
My global.json config is the followings.
```
{
  "sdk": {
    "version": "8.0.100",
    "allowPrerelease": true,
    "rollForward": "major"
  },
  "tools": {
    "dotnet": "8.0.100"
  },
  "msbuild-sdks": {
    "Microsoft.DotNet.Arcade.Sdk": "9.0.0-beta.23578.2",
    "Microsoft.DotNet.Helix.Sdk": "9.0.0-beta.23578.2",
    "Microsoft.DotNet.SharedFramework.Sdk": "9.0.0-beta.23578.2",
    "Microsoft.Build.NoTargets": "3.7.0",
    "Microsoft.Build.Traversal": "3.4.0",
    "Microsoft.NET.Sdk.IL": "9.0.0-alpha.1.23576.6"
  }
}
```
How can I build aspnetcore for net8.0?

BR,

**Comment by davidfowl** [comment, marked as answer]
There's a 8.0 branch:

https://github.com/dotnet/aspnetcore/tree/release/8.0

**Reply by corsiva** [reply]
Thanks for the help.
You made me think about branch.
The issue was that I've created new branch, instead of switching to 8.0 branch.

BR,
